packages/dx-grid-core/src/plugins/table-column-resizing/reducers.ts

```diff
@@ -17,11 +32,26 @@ export const changeTableColumnWidth: ColumnWidthReducer = (
 };
 
 export const draftTableColumnWidth: ColumnWidthReducer = (
-  state, { columnName, shift, minColumnWidth },
+  state, { columnName, shift, minColumnWidth, maxColumnWidth, columnExtensions },
 ) => {
   const { columnWidths } = state;
   const updatedColumn = columnWidths.find(elem => elem.columnName === columnName)!;
-  const size = Math.max(minColumnWidth, updatedColumn.width + shift);
+  const indexOfExtended = columnExtensions
```

Comment:
`changeTableColumnWidth` and `draftTableColumnWidth` should be refactored, repeated code should be extracted